conf/services/get-a-goods-movement-reference.cy.yml

```diff
@@ -6,10 +6,10 @@ serviceDomain: www.tax.service.gov.uk
 serviceUrl: /goods-movement-system/dashboard
 contactFrontendServiceId: GaGMR
 complianceStatus: full
-serviceLastTestedDate: 2022-03-22
+serviceLastTestedDate: 2022-07-18
 statementVisibility: public
 statementCreatedDate: 2020-11-16
-statementLastUpdatedDate: 2023-06-05
+statementLastUpdatedDate: 2023-07-24
```

Comment:
tomorrow 25th is more feasible date for this to go live

Reply:
usually plat-ui are quick they do the deployment on the same day,  you mean we are deploying all our a11y fixes tomorrow or any other reason for tomorrow ?

Reply:
i am happy to use 25th but if that team releases these changes today, then there is a slight chance that people today see future date (25th) as last updated

Reply:
compliance is not changing from the previous version, so I think it is ok to go ahead.